business_service.go

```diff
@@ -49,12 +49,23 @@ type ListBusinessServiceOptions struct {
 
 // ListBusinessServices lists existing business services.
 func (c *Client) ListBusinessServices(o ListBusinessServiceOptions) (*ListBusinessServicesResponse, error) {
+	bss, err := c.ListBusinessServicesPaginated(context.Background(), o)
+	if err != nil {
+		return nil, err
+	}
+
+	return &ListBusinessServicesResponse{BusinessServices: bss}, nil
+}
+
+// ListBusinessServicesPaginated lists existing business services, automatically
+// handling pagination and returning the full collection.
+func (c *Client) ListBusinessServicesPaginated(ctx context.Context, o ListBusinessServiceOptions) ([]*BusinessService, error) {
```

Comment:
This takes naming inspiration from #295.

Reply:
Good call.